src/readqps.jl

```diff
@@ -800,7 +808,18 @@ function readqps(qps::IO; mpsformat::Symbol=:free)
         elseif sec == ROWS
             read_rows_line!(qpsdat, card)
         elseif sec == COLUMNS
-            read_columns_line!(qpsdat, card)
+            # Check if card is marker
+            if card.f2 == "'MARKER'" 
+                if card.f3 == "'INTORG'"
+                    integer_section = true
+                elseif card.f3 == "'INTEND'"
+                    integer_section = false
+                else
+                    @error "Ignoring marker $(card.f3) at line $(card.nline)"
```

Comment:
Shouldn't this just be `@warn`?

Reply:
The convention we adopted is that rim data trigger an `@error` log message.
Such messages can be suppressed by setting the logger as desired.

Reply:
Ohhhh. I was wondering why I didn't see the errors when reading SMIPs. I forgot about the `NullLogger` in SMPSReader.